heimdall-core/src/main/java/br/com/conductor/heimdall/core/repository/jdbc/InterceptorJDBCRepository.java

```diff
@@ -36,23 +36,26 @@
 
     private JdbcTemplate jdbcTemplate;
 
+    private static final String SQL = "SELECT I.ID, I.API_ID, I.EXECUTION_POINT, I.LIFE_CYCLE, I.TYPE, I.NAME, I.CONTENT, I.EXECUTION_ORDER, I.STATUS, " +
```

Comment:
SQL is not a good name for a variable. Change to a name that makes more sense

Reply:
Changed to LIST_INTERCEPTOR